fedimint-testing/src/gateway.rs

```diff
@@ -73,11 +84,17 @@ impl GatewayTest {
         })
         .await;
 
-        Self {
+        let mut gateway = Self {
+            feds: vec![],
             password,
-            address,
+            api: address,
             _config_dir,
```

Comment:
Can these just be deleted if they're not used?

Reply:
The config dir we need to hold a handle to otherwise it'll drop which auto deletes the config dirs.